tutorials/preprocessing/plot_15_handling_bad_channels.py

```diff
@@ -0,0 +1,209 @@
+# of magnitude greater than the other MEG channels. Regardless of what is
+# causing the problems, neither are realistic reflections of the
+# electrophysiological signals we expect, so it makes sense to exclude them.
+#
+# If you want to change which channels are marked as bad, you can edit
+# ``raw.info['bads']`` directly; it's an ordinary Python :class:`list` so the
+# usual list methods will work:
+
+original_bads = raw.info['bads']
+raw.info['bads'].append('EEG 050')               # add a single channel
+raw.info['bads'].extend(['EEG 051', 'EEG 052'])  # add a list of channels
+bad_chan = raw.info['bads'].pop(-1)  # remove the last entry in the list
+raw.info['bads'] = original_bads     # change the whole list at once
+
+###############################################################################
+# .. sidebar:: Blocking execution
+#
+#     If you want to build an interactive bad-channel-marking step into an
+#     analysis script, be sure to include the parameter ``block=True`` in your
+#     call to ``raw.plot()`` or ``epochs.plot()``. This will pause the script
+#     while the plot is open, giving you time to mark bad channels before
+#     subsequent analysis or plotting steps are executed.
+#
+# You can also interactively toggle whether a channel is marked "bad" in the
+# plot windows of ``raw.plot()`` or ``epochs.plot()`` by clicking on the
+# channel name along the vertical axis (in ``raw.plot()`` windows you can also
+# do this by clicking the channel's trace in the plot area). The ``bads`` field
+# gets updated immediately each time you toggle a channel, and will retain its
+# modified state after the plot window is closed.
+#
+# The list of bad channels in the :class:`mne.Info` object's ``bads`` field is
+# automatically taken into account in dozens of functions and methods across
+# the MNE-Python codebase. This is done consistently with a parameter
+# ``exclude='bads'`` in the function or method signature. Typically this
+# ``exclude`` parameter also accepts a list of channel names or indices, so if
+# you want to *include* the bad channels you can do so by passing
+# ``exclude=[]`` (or some other list of channels to exclude).
+#
+#
+# When to look for bad channels
+# ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+#
+# You can start looking for bad channels during the experiment session when the
+# data is being acquired. If you notice any flat or excessively noisy channels,
+# you can note them in your experiment log or protocol sheet. If your system
+# computes on-line averages, these can be a good way to spot bad channels as
+# well. After the data has been collected, you can do a more thorough check for
+# bad channels by browsing the raw data using :meth:`mne.io.Raw.plot`, with any
+# projectors or ICA applied. Finally, you can compute off-line averages (again
+# with projectors, ICA, and EEG referencing disabled) to look for channels with
+# unusual properties.
+#
```

Comment:
We could actually show an ERF/P plot where bad channels are not marked. It is a fantastic way to spot issues.